### PR TITLE
fetching username and access key from .env for triggering test only

### PR DESCRIPTION
**Pull request by HRanjan-11**

### CODE REVIEW

File: commands/utils/set_args.js [hunk 1]
```diff
@@ -18,6 +18,51 @@ function sync_args_from_cmd(args) {
...
+    let dot_env_vars = undefined;
+    let envFilePath = path.join(".", `.env`);
+    if ("sys-env-keys" in args) {
+      dot_env_vars = args["sys-env-keys"];
+    } else if (lt_config["run_settings"] && lt_config["run_settings"]["sys_env_keys"]) {
+      dot_env_vars = lt_config["run_settings"]["sys_env_keys"];
+    }
+    if (dot_env_vars) {
+      dot_env_vars = dot_env_vars.trim();
+      dot_env_vars = dot_env_vars.split(",");
```
Comment (shubhamlt) [inline commands/utils/set_args.js:33]: lets use a diff var

File: commands/utils/set_args.js [hunk 2]
```diff
@@ -18,6 +18,51 @@ function sync_args_from_cmd(args) {
...
+      if ("envfl" in args) {
+        envFilePath = args["envfl"];
+      } else if (lt_config["run_settings"]["env_file"]) {
```
Comment (shubhamlt) [inline commands/utils/set_args.js:36]: lt_config["run_settings"] && lt_config["run_settings"]["env_file"]

**Review by shubhamlt — CHANGES_REQUESTED** [review]
pl do the commented changes

File: commands/utils/set_args.js [hunk 3]
```diff
@@ -18,6 +18,51 @@ function sync_args_from_cmd(args) {
   return new Promise(function (resolve, reject) {
     let rawdata = fs.readFileSync(args["lambdatest-config-file"]);
     let lt_config = JSON.parse(rawdata);
```
[the remark this answers is not in the document]
Comment (HRanjan-11) [reply commands/utils/set_args.js]: done

File: commands/utils/set_args.js (hunk 1 above)
Comment (HRanjan-11) [reply commands/utils/set_args.js]: done

File: commands/utils/set_args.js (hunk 2 above)
Comment (HRanjan-11) [reply commands/utils/set_args.js]: done

**Pull request merged**